rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/LoggersWindow.java

```diff
@@ -0,0 +1,207 @@
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive_grid_cloud_portal.rm.client;
+
+import org.ow2.proactive_grid_cloud_portal.common.client.Images;
+
+import com.smartgwt.client.types.Alignment;
+import com.smartgwt.client.types.ListGridFieldType;
+import com.smartgwt.client.widgets.IButton;
+import com.smartgwt.client.widgets.Label;
+import com.smartgwt.client.widgets.Window;
+import com.smartgwt.client.widgets.events.ClickEvent;
+import com.smartgwt.client.widgets.events.ClickHandler;
+import com.smartgwt.client.widgets.events.CloseClickEvent;
+import com.smartgwt.client.widgets.events.CloseClickHandler;
+import com.smartgwt.client.widgets.grid.ListGrid;
+import com.smartgwt.client.widgets.grid.ListGridField;
+import com.smartgwt.client.widgets.grid.ListGridRecord;
+import com.smartgwt.client.widgets.layout.HLayout;
+import com.smartgwt.client.widgets.layout.Layout;
+import com.smartgwt.client.widgets.layout.VLayout;
+
+
+/**
+ * Client-side Loggers settings menu
+ *
+ * Changes are effective immediately
+ *
+ * Changes or newly added records are made in memory and not persisted
+ *
+ */
+
+public class LoggersWindow {
+
+    private Window window;
+
+    private RMController controller;
+
+    private ListGrid loggersGrid;
+
+    public LoggersWindow(RMController controller) {
+        this.controller = controller;
+        this.build();
+    }
+
+    public void show() {
+        this.window.show();
+    }
+
+    public void destroy() {
+        this.window.destroy();
+    }
+
+    private void build() {
+        Layout layout = new VLayout();
+        layout.setMembersMargin(15);
+        layout.setAutoHeight();
+        layout.setMargin(10);
+
+        Label label = new Label("Edit existing loggers levels and/or add a new logger/level. Please note that changes are made in memory and not persisted.");
+        label.setHeight(30);
+
+        loggersGrid = new ListGrid();
+        loggersGrid.setWidth(500);
+        loggersGrid.setHeight(244);
+        loggersGrid.setAlternateRecordStyles(true);
+        loggersGrid.setShowAllRecords(true);
+
+        ListGridField loggerField = new ListGridField("logger", "Logger", 385);
+        loggerField.setRequired(true);
+
+        ListGridField levelField = new ListGridField("level", "Level");
+        levelField.setValueMap("TRACE", "DEBUG", "INFO", "WARN", "ERROR", "FATAL", "OFF");
+        levelField.setCanEdit(true);
+
+        ListGridField removeField = new ListGridField(" ", 20);
+        removeField.setType(ListGridFieldType.IMAGE);
+        removeField.setCanEdit(false);
+        removeField.setCanSort(false);
+        removeField.setDefaultValue("blank.png");
+
+        loggersGrid.setFields(new ListGridField[] { loggerField, levelField, removeField });
```

Comment:
No need to create an array here, you can pass as varargs.

Reply:
True, I fixed it !